server/backend/api/app/crud/crud_application.py

```diff
@@ -189,6 +189,9 @@ def get_application_role_assignments(
 
 
 def get_application_id_by_role_id(db: Session, role_id):
+    # question: why we need to go to fam_application table?
+    # the fam_role table has the application_id already
+
     application_id =\
         db.query(models.FamApplication.application_id)\
```

Comment:
Got a question here, why we need to go to fam_application table? The fam_role table has the application_id already

Reply:
Seems to me this query can be improved; but we can touch that next time doing implementation I think.